src/requirements-dev.txt

```diff
@@ -2,7 +2,8 @@ black==24.1.1
 coverage==7.4.1
 flake8==7.0.0
 isort==5.13.2
-pytest==8.0.0
+pytest==8.2.0
```

Comment:
next PR will be other python updates so I'll put the version upgrade in that one